Input/input.html

```diff
@@ -0,0 +1,176 @@
+              Badge
+              </a>
+          </li>
+          <li>
+              <a href="../Button/button.html" class="sidebar-menu" rel="noopener noreferrer">
+              Button
+              </a>
+          </li>
+          <li>
+              <a href="../Card/card.html" class="sidebar-menu" rel="noopener noreferrer">
+              Card
+              </a>
+          </li>
+          <li>
+              <a href="../Image/image.html" class="sidebar-menu" rel="noopener noreferrer">
+              Image
+              </a>
+          </li>
+          <li>
+              <a href="../Input/input.html" class="sidebar-menu" rel="noopener noreferrer">
+              Input
+              </a>
+          </li>
+          <li>
+              <a href="../TextUtilities/textUtilities.html" class="sidebar-menu" rel="noopener noreferrer">
+              Text Utilities
+              </a>
+          </li>
+          <li>
+              <a href="../List/list.html" class="sidebar-menu" rel="noopener noreferrer">
+              List
+              </a>
+          </li>
+          <li>
+              <a href="../Navigation/navigation.html" class="sidebar-menu" rel="noopener noreferrer">
+              Navigation
+              </a>
+          </li>
+          <li>
+              <a href="../Modal/modal.html" class="sidebar-menu" rel="noopener noreferrer">
+              Modal
+              </a>
+          </li>
+          <li>
+              <a href="../Rating/rating.html" class="sidebar-menu" rel="noopener noreferrer">
+              Rating
+              </a>
+          </li>
+          <li>
+              <a href="../Toast/toast.html" class="sidebar-menu" rel="noopener noreferrer">
+              Toast
+              </a>
+          </li>
+          <li>
+              <a href="../SimplifiedGrid/grid.html" class="sidebar-menu" rel="noopener noreferrer">
+              Simplified Grid
+              </a>
+          </li>
+      </ul>
+      </div>
+      <div class="main">
+        <h1 class="font-xl margin-m2">Input</h1>
+        <br />
+        <hr />
+        <h3 class="font-medium margin-m2">Primary Inputs With Textbox</h3>
+        <p  class="font-medium label-text"> Primary input with textbox is one way of taking input from the user.</p>
+
+        <div class="box container-flex flex-column">
+          <textarea
+            class="text-area"
+            cols="300"
+            rows="10"
+            placeholder="Enter a text"
+          >
+          </textarea>
+          <br />
+          <br />
+          <input type="text" class="input" placeholder="Enter text" /> <br />
+          <br />
+          <input type="number" class="input" placeholder="Enter Number" />
+          <br />
+          <br />
+          <input type="password" class="input" placeholder="Enter password" />
+          <br />
+          <br />
+        </div>
+        <div class="margin-m2 padding-p2">
```

Comment:
instead of margin-m2 and padding-p2 you can give shorter names for space classes from next time.

Reply:
definitely